docs/source/en/model_doc/mobilenet_v1.md

```diff
@@ -14,43 +14,102 @@ rendered properly in your Markdown viewer.
+print(predicted_label)
+# Example output: tabby, tabby cat
+```
 
-- The original TensorFlow checkpoints use different padding rules than PyTorch, requiring the model to determine the padding amount at inference time, since this depends on the input image size. To use native PyTorch padding behavior, create a [`MobileNetV1Config`] with `tf_padding = False`.
+</hfoption>
+</hfoptions>
 
-Unsupported features:
+<!-- Quantization - Not applicable -->
+<!-- Attention Visualization - Not applicable for this model type -->
 
-- The [`MobileNetV1Model`] outputs a globally pooled version of the last hidden state. In the original model it is possible to use a 7x7 average pooling layer with stride 2 instead of global pooling. For larger inputs, this gives a pooled output that is larger than 1x1 pixel. The HuggingFace implementation does not support this.
 
-- It is currently not possible to specify an `output_stride`. For smaller output strides, the original model invokes dilated convolution to prevent the spatial resolution from being reduced further. The output stride of the HuggingFace model is always 32.
+## Notes
 
-- The original TensorFlow checkpoints include quantized models. We do not support these models as they include additional "FakeQuantization" operations to unquantize the weights.
+-   **Checkpoint Naming:** Checkpoints often follow the pattern `mobilenet_v1_{depth_multiplier}_{resolution}`, like `mobilenet_v1_1.0_224`.
+-   **Variable Input Size:** While trained on fixed sizes (e.g., 224x224), the model architecture works with images of different sizes (minimum 32x32). The [`MobileNetV1ImageProcessor`] handles the necessary preprocessing.
+-   **1001 Classes:** Models pretrained on ImageNet-1k output 1001 classes. Index 0 is a "background" class, and indices 1-1000 correspond to the standard ImageNet classes.
+-   **Padding Differences:** Original TensorFlow checkpoints had dynamic padding behavior based on input size. The Hugging Face PyTorch implementation uses standard, static padding by default. To enable dynamic padding (matching TensorFlow behavior), instantiate the configuration with `tf_padding=True`.
+    ```python
+    from transformers import MobileNetV1Config
 
-- It's common to extract the output from the pointwise layers at indices 5, 11, 12, 13 for downstream purposes. Using `output_hidden_states=True` returns the output from all intermediate layers. There is currently no way to limit this to specific layers.
+    # Example: Load config with dynamic padding enabled
+    config = MobileNetV1Config.from_pretrained("google/mobilenet_v1_1.0_224", tf_padding=True)
+    ```
+-   **Unsupported Features:** The Hugging Face implementation has some differences from the original paper/TF implementation:
+    -   Uses global average pooling instead of the optional 7x7 average pooling with stride 2.
+    -   Does not support specifying a custom `output_stride` (which would require dilated convolutions). The output stride is fixed at 32.
```

Comment:
```suggestion
    -   Does not support other `output_stride` values (fixed at 32). For smaller `output_strides`, the original implementation uses dilated convolution to prevent spatial resolution from being reduced further. (which would require dilated convolutions).
```